Cargo.toml

```diff
@@ -100,7 +100,7 @@ use_self = "warn"
 ## By default, we enable the crossterm backend as this is a reasonable choice for most applications
 ## as it is supported on Linux/Mac/Windows systems. We also enable the `underline-color` feature
 ## which allows you to set the underline color of text.
-default = ["crossterm", "underline-color"]
+default = ["termion", "underline-color"]
```

Comment:
This was probably changed while working on it and mistakenly added to the commit?

Reply:
Oh, nice catch. Fixed.